services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -62,18 +63,18 @@ open class HankeServiceImpl(
     private val applicationService: ApplicationService,
 ) : HankeService {
 
-    override fun getHankeId(hankeTunnus: String): Int? =
-        hankeRepository.findByHankeTunnus(hankeTunnus)?.id
+    override fun getHankeId(hankeTunnus: String?): Int? =
```

Comment:
Calling this with a null parameter makes no sense. I think the caller should handle the situations where hankeTunnus might be null.

Reply:
This is now irrelevant as we kept Application hanketunnus non nullable. Fixed.